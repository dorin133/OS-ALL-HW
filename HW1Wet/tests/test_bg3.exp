smash> smash error: bg: there is no stopped jobs to resume
smash> smash> smash error: bg: there is no stopped jobs to resume
smash> smash> [1] sleep 100& : 111003 0 secs
[2] sleep 200& : 111004 0 secs
smash> smash error: bg: job-id 2 is already running in the background
smash> smash: got ctrl-Z
smash: process 111005 was stopped
smash> sleep 300 : 111005
smash> smash error: bg: job-id 3 is already running in the background
smash> smash: sending SIGKILL signal to 3 jobs:
111003: sleep 100&
111004: sleep 200&
111005: sleep 300
