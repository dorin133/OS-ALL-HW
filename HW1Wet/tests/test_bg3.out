smash> smash error: bg: there is no stopped jobs to resume
smash> smash> smash error: bg: there is no stopped jobs to resume
smash> smash> [1] sleep 100& : 111003 0 secs
[2] sleep 200& : 111005 0 secs
smash> smash error: bg: job-id 2 is already running in the background
smash> smash: got ctrl-Z
smash: process 111009 was stopped
smash> sleep 300 : 111009 
smash> smash error: bg: job-id 3 is already running in the background
smash> smash: sending SIGKILL signal to 3 jobs:
111003: sleep 100&
111005: sleep 200&
111009: sleep 300
